react/README.md

```diff
@@ -26,48 +27,28 @@ This guide provides a few guidelines on writing sensible React. Many of these ru
 
   ```js
   // bad
-  class financialReport extends React.Component {}
+  class financialReport extends Component {}
```

Comment:
why this change? The react guide does `React.Component`, I think we should just follow that by default https://facebook.github.io/react/docs/reusable-components.html#es6-classes